sections/admin/clients/new-client.jsx

```diff
@@ -16,8 +17,33 @@ const AdminNewClient = () => {
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    toast.success("Успішно");
-    setTimeout(() => Router.push("/admin/clients"), 500);
+    const dataNewClient = { name, carBrand, phoneNumber, carModel, email, licensePlate };
```

Comment:
we need to add check if every field is empty before submit. For instance if we dont pass `email` (it is not required) it will get in dataNewClient with empty string "" and back-end will throw the error `"email" is not allowed to be empty`

Reply:
![image](https://user-images.githubusercontent.com/84677177/191317250-28f6192f-3d3d-4183-bef1-dbf15040e60f.png)

Reply:
Okay, agree.